### PR TITLE
Systemd-based test container + tests for Sysbox systemd cgroup v1 support 

### PR DESCRIPTION
**Pull request by ctalledo**

### CODE REVIEW

File: tests/scr/testContainerInit [hunk 1]
```diff
@@ -49,120 +42,143 @@ function retry() {
...
+   # Identify default egress iface.
+   local egress_iface=$(ip route show | awk '/default via/ {print $5}')
+   if [ -z "${egress_iface}" ]; then
+		return
+   fi
+
+   # Obtain mtu value associated to the test-container's egress interface.
+   egress_mtu=$(ip link show ${egress_iface} | awk '/mtu / {print $5}')
+   if [ ! -z "${egress_mtu}" ] &&
+			[ "${egress_mtu}" -lt "${default_mtu}" ]; then
```
Comment (rodnymolina) [inline tests/scr/testContainerInit:58]: Check indentation.

**Review by rodnymolina — APPROVED** [review]
Looks great. Have only a couple of comments to look at, all the others are mainly indentation issues to fix.

File: scr/docker-cfg [hunk 2]
```diff
@@ -0,0 +1,554 @@
...
+    # If the dockerd default config-file exist, copy it to a temp file; we
+    # operate on the temp file and copy it back to the real config file at the
+    # end (unless --dry-run is set).
+	if [[ -f ${dockerCfgFile} ]]; then
+		cp ${dockerCfgFile} ${tmpCfgFile}
+	else
+		touch ${tmpCfgFile}
+	fi
+
+    # If there's no content on it, create one with a bare json layout.
```
[the remark this answers is not in the document]
Comment (ctalledo) [reply scr/docker-cfg:396]: Fixed.

File: scr/docker-cfg [hunk 3]
```diff
@@ -0,0 +1,554 @@
...
+        skip_network_config=1
+        ;;
+      --dry-run)
+        dry_run=1
+        ;;
+      --sysbox-runtime)
+        shift;
+        sysbox_runtime=$1
+        if [[ ${sysbox_runtime} != "enable" ]] && [[ ${sysbox_runtime} != "disable" ]]; then
+			  printf "\nInvalid value for sysbox-runtime.\n"
```
[the remark this answers is not in the document]
Comment (ctalledo) [reply scr/docker-cfg:487]: Fixed.

File: tests/scr/testContainerInit (hunk 1 above)
Comment (ctalledo) [reply tests/scr/testContainerInit:58]: Fixed.

**Pull request merged**